leptonai/cli/job.py

```diff
@@ -28,6 +28,24 @@
 from leptonai.api.v1.client import APIClient
 
 
+def _get_newest_job_by_name(job_name: str) -> LeptonJob:
+    client = APIClient()
+    job_list = client.job.list_all()
+    cur_job_list = []
+    for job in job_list:
+        if job.metadata.name == job_name:
+            cur_job_list.append(job)
+
+    if len(cur_job_list) == 0:
+        return None
+
+    jobs_sorted_descending = sorted(
```

Comment:
the variable named `jobs_sorted_descending` but it's not sorted by descending, lol